hack/externalnode/install-vm.sh

```diff
@@ -40,50 +42,53 @@ function print_help {
 }
 
 INSTALL_PATH="/usr/sbin"
-AGENT_BIN_PATH=""
-CONFIG_PATH=""
-KUBECONFIG=""
-ANTREAKUBECONFIG=""
-AGENT_NAMESPACE=""
-NODE_NAME="$(hostname)"
+ANTREA_AGENT="antrea-agent"
+ANTREA_CLEANUP="antrea-cleanup"
```

Comment:
Not consumed?

Reply:
Left over from my earlier PR. will remove it.